src/neo-vm/OpCode.cs

```diff
@@ -225,9 +225,11 @@ public enum OpCode : byte
         /// Pop the address of a function from the stack, and call the function.
         /// </summary>
         CALLA = 0x36,
-        THROW = 0x37,
-        THROWIF = 0x38,
-        THROWIFNOT = 0x39,
+        /// <summary>
+        /// Pop the top value of the stack, if it false, then exit vm execution.
+        /// </summary>
+        ASSERT = 0x38,
```

Comment:
We should insert a new opcode `FAULT` at position 0x37. It tures the vm state to `FAULT` immediately, and cannot be caught.

Reply:
`ABORT` or `EXIT` will be better? Or don't need to add `FAULT`, which equals `ASSERT(false)`?

Reply:
Personally, I don't think we need so many "unstable" instructions, also as @shargon said. 

> Because is something easier to implement and extend in the future, VM should be more stable

Reply:
I like `ABORT`. If you have `TRY` and `CATCH`, you should need an `ABORT`. Otherwise one can just write a big try and ignore all the exceptions.

Reply:
Got, it makes sense.